### DISCUSSION TITLE
Freezing on micro motherboards

### DISCUSSION THREAD

**Discussion by karstengit** [memtest86plus/memtest86plus#119] [Q&A]
This is more an support question as an bug.

I want to test the RAM (8 GB) of two older micro mainboards with a Phenom II x4 CPU.
First i thought that the RAM is defective, but then i found this hints: https://www.memtest86.com/tech_freezing-lockups.html

This leads to three questions:

a) What can alternatively used / be done when a RAM-Test fails with memtest86+ ?

b) How it is possible to distinguish between BIOS-Bugs and bugs in the multithread support ?

c) I am estonished that the BIOS can cause CPU freeze and/or RAM problems.
Of course it is clear that the BIOS is setting the speed and voltage for the Dimm-modules at boot up.
But afterwards there should be no other BIOS functionality, or can someone please explain why here the BIOS is "needed" ?
I assume memtest86+ does not need any kernel or other "higher functionality", so that it can be plain executed code ?
From my understanding there should be nothing standing between a CPU and the RAM.
What is going on in the above cases of freezing?

When more informations are needed, i must rerun the tests.
Last i used memtest86+ with booting from an USB-Stick containing grub and an recent version of memtest86+.

**Comment by x86fr** [comment]
To investigate an issue, we need to know 

1. the exact model of the motherboard 
2. the version of Memtest86+ used
3. a precise description of the issue 

Memtest86+ does not work the same way as the commercial, closed-source "Memtest86". The later is an UEFI application relying on the UEFI framework while Memtest86+ is a stand-alone application.

A bugged BIOS can easily cause severe issues in various different way. Ie: the BIOS is in charge of populating the memory map. If it wrongly marks reserved memory as free/usable, the PC might crash as soon as this part of memory will be accessed.

**Reply by karstengit** [reply]
Must the provided functionality of the BIOS be used to run a memory test, that is the only application running on the system?
